spec/facades/food_facade_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'Food Facade' do
+  it 'returns data for the searched food' do
+    data = FoodFacade.first_10('sweet potatoes')
+
```

Comment:
These are great tests as well. I would just return the data types. Also I would consider using vcr or webmock for stubbing